frameworks/kafka/docs/upgrade.md

```diff
@@ -38,24 +38,24 @@ This is an overview of the rolling upgrade process. The process for upgrading Ka
 
 ### Detailed upgrade instructions
 
-The new DC/OS Kafka package `2.0.0-0.11.0.0` (with Kafka version 0.11.0.0) that we will be upgrading to has the following default settings:
+The new DC/OS Kafka package `1.1.27-0.11.0-beta` (with Kafka version 0.11.0.0) that we will be upgrading to has the following default settings:
```

Comment:
The new DC/OS Kafka package `1.1.27-0.11.0-beta` (with Kafka version 0.11.0.0) has the following default settings